src/renderer/apps/multisend/def.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import {ReactComponent as MultisendIcon} from "./icon.svg"
+
+import { AppDefinition } from "../../components/app-definition"
+import MultiSendApp from "./multisend"
+import SvgIcon from '@material-ui/core/SvgIcon';
+
+const CVIcon = (props: {fontSize?: "default" | "large"}): JSX.Element => {
+	return <SvgIcon
+		fontSize={props.fontSize}
+		component={MultisendIcon}
+		/>
+}
+
+export const MultiSend: AppDefinition = {
+	id: "multisend",
+	title: "MultiSend",
+	icon: <CVIcon />,
+	iconLarge: <CVIcon fontSize="large" />,
+	core: true,
```

Comment:
This isn't a core app? Core apps are those that come pre-installed. This is just a regular installable app.